src/components/CreateFund/CreateFundForm.tsx

```diff
@@ -9,7 +9,7 @@ import { Trans, t } from "@lingui/macro";
 import Error from "../ui/Error";
 import { Link } from "react-router-dom";
 import { Input } from "../Form/Input";
-import Button from "../Form/Button";
+import Button from "../ui/Button";
```

Comment:
you want to move the button because it's not only used for Forms right? then just do Button/Button so we can put all kinds of buttons in there. A folder called `ui` isnt very helpful because the entire app is UI

Reply:
Yeah got it, That's a very helpful suggestion, thanks.